docs/content/v0.20/README.md

```diff
@@ -42,4 +42,5 @@ See [Packaging and Delivery](packaging.md)
 
 ## Usage examples, and testing
 
-See [Examples](examples.md).
+See [Examples](examples.md) and [integration testing](contributor.md#integration-testing).
```

Comment:
Why explicitly mention integration tests and not unit test?

Reply:
I did not want to increase the scope of this PR more than necessary. Unit testing is almost completely missing, and is completely undocumented. This PR does not add or remove unit tests.